packages/lib/src/components/results/ResultTableComponent.wc.svelte

```diff
@@ -155,47 +133,46 @@
         }
     };
 
-    /**
-     * sort tableRowData alphanumerically by the given column
-     */
-
-    let sortColumnIndex: number = 0;
+    export let pageSize = 10;
+    let activePage = 1;
+    let sortColumnIndex = 0;
+    let sortAscending = true;
 
-    /**
-     * sorts the tableRowData by the given column
-     * @param column - column to sort
-     * @param ascending - order of the sort, changes after every click but not on incoming responses
-     * @param tableRowData - as an argument to make the function reactive and prevent race conditions with incoming responses
-     * @param changeAscending - if true, the order of the sort will change after every click
-     * @returns the sorted tableRowData
-     */
-    const sortTable = (
-        column: number,
-        ascending: boolean = true,
-        tableRowData: TableRowData,
-        changeAscending: boolean = false,
-    ): TableRowData => {
-        /**
-         * sets the index of the column to sort, so that further incoming responses don't mess up the sorting
-         */
-        sortColumnIndex = column;
+    let visibleRows: TableRowData;
+    $: {
+        // Array.sort sorts in place, so make a copy first
+        const tableRowsCopy = [...tableRowData];
 
-        tableRowData = tableRowData.sort((a, b) => {
-            if (a[column] < b[column]) {
-                return ascending ? -1 : 1;
+        // sort
+        tableRowsCopy.sort((a, b) => {
+            if (a[sortColumnIndex] < b[sortColumnIndex]) {
```

Comment:
While some sites are still processing, they don't get sorted. If you sort by a number, eg. patient count, the processing should always be at the end